myhoard/controller.py

```diff
@@ -1163,13 +1172,28 @@ def _purge_old_binlogs(self, *, mysql_maybe_not_running=False):
                 last_purge = time.time()
                 last_could_have_purged = last_purge
         finally:
+            current_time = time.time()
+            
             self.state_manager.update_state(
-                binlogs_purged_at=time.time(),
+                binlogs_purged_at=current_time,
                 last_binlog_purge=last_purge,
                 last_could_have_purged=last_could_have_purged,
             )
-            self.stats.gauge_float("myhoard.binlog.time_since_any_purged", time.time() - last_purge)
-            self.stats.gauge_float("myhoard.binlog.time_since_could_have_purged", time.time() - last_could_have_purged)
+
+            self.stats.gauge_float("myhoard.binlog.time_since_any_purged", current_time - last_purge)
+            self.stats.gauge_float(
+                "myhoard.binlog.time_since_could_have_purged", 
+                current_time - last_could_have_purged,
+                tags = { "support_oldest_should_have_purged" : "1" }
```

Comment:
We're apparently not running any style checkers for this repo but should be written as `tags={"support_oldest_should_have_purged": "1"}`